CIP-0036/test-vector.md

```diff
@@ -1,110 +1,135 @@
-# Test vector for CIP15
+# Test Vector for CIP-0036
 
-### Inputs
+## Keys
 
-Payment **public** key hex
+**Note:** Not all keys are required for certificate recreation.
+
+Payment **private** signing key cborHex:
 ```
-3273a5316e4de228863bd7cf8dac90d57149e1a595f3dd131073b84e35546676
+5820614fdfe13d403bee2014570b190d81851f17d8daca0b6dd1ce33014403191003
```

Comment:
There's an extra CBOR wrapping on this key. It should be:

```
614fdfe13d403bee2014570b190d81851f17d8daca0b6dd1ce33014403191003
```

(a private key is 32-byte long).